dietpi/dietpi-software

```diff
@@ -11723,43 +11746,14 @@ _EOF_
 			# On fresh install, pre-configure for Sonarr and Radarr when found
 			if [[ ! -f '/mnt/dietpi_userdata/bazarr/config/config.ini' ]] && (( ${aSOFTWARE_INSTALL_STATE[144]} + ${aSOFTWARE_INSTALL_STATE[145]} > 0 ))
 			then
-				local timeout=15
-				(( $G_HW_ARCH > 2 )) || timeout=20
-				G_EXEC_DESC='Starting Bazarr to pre-create config file and pre-configure Sonarr and/or Radarr connection' G_EXEC systemctl start bazarr
-				local i=0
-				until [[ -f '/mnt/dietpi_userdata/bazarr/config/config.ini' || $i -ge $timeout ]]
-				do
-					((i++))
-					G_DIETPI-NOTIFY -2 "Waiting for Bazarr config file to be created ($i/$timeout)"
-					sleep 1
-				done
-				sleep 1
-				G_EXEC_NOHALT=1 G_EXEC systemctl stop bazarr
-				sleep 1
-				if [[ ! -f '/mnt/dietpi_userdata/bazarr/config/config.ini' ]]
-				then
-					G_DIETPI-NOTIFY 1 'Waiting for Bazarr config file to be created timed out, skipping Sonarr and Radarr pre-configuration'
+				CONFIG_CREATE /mnt/dietpi_userdata/bazarr/config/config.ini bazarr
 				else
```

Comment:
The else here is wrong now, also the opening if is missing. With an explicit return code (see above), we could do:
```
if CONFIG_CREATE /mnt/dietpi_userdata/bazarr/config/config.ini bazarr
then
```